worker/src/main/java/gov/cms/ab2d/worker/service/WorkerServiceImpl.java

```diff
@@ -16,17 +21,37 @@
 
     private final JobPreProcessor jobPreprocessor;
     private final JobProcessor jobProcessor;
+    private final ShutDownService shutDownService;
+
+    private static List<String> activeJobs = Collections.synchronizedList(new ArrayList<>());
 
 
     @Override
     public void process(String jobUuid) {
+
+        activeJobs.add(jobUuid);
+
         jobPreprocessor.preprocess(jobUuid);
         log.info("Job was put in progress");
 
         jobProcessor.process(jobUuid);
         log.info("Job was processed");
+
+        activeJobs.remove(jobUuid);
```

Comment:
We need a `try/finally` block here and remove the `jobUuid` from the list in the `finally` clause.

Reply:
done